lib/settlement/null-settlement-contract.js

```diff
@@ -1,25 +1,11 @@
 'use strict';
 
-const ethers = require('ethers');
+const NahmiiContract = require('../contract');
 
-class NullSettlementContract extends ethers.Contract {
+class NullSettlementStateContract extends NahmiiContract {
```

Comment:
Namechange ?

Reply:
Waiting for feedback from @katat  - I cant see this class/file actually being used anywhere so might just remove it.

Reply:
Reverted namechange